src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions.test.ts

```diff
@@ -1098,9 +1096,7 @@ describe('migration actions', () => {
         _tag: 'Left',
         left: {
           error: expect.any(errors.ResponseError),
-          message: expect.stringContaining(
-            '[timeout_exception] Timed out waiting for completion of task'
-          ),
+          message: expect.stringContaining('[timeout_exception]'),
```

Comment:
That's at least the second (third?) time the message returned by ES changes. I changed the assertion to just check about the error name now...